src/java/org/apache/nutch/indexer/IndexerMapReduce.java

```diff
@@ -70,51 +68,26 @@
   public static final String URL_NORMALIZING = "indexer.url.normalizers";
   public static final String INDEXER_BINARY_AS_BASE64 = "indexer.binary.base64";
 
-  private boolean skip = false;
-  private boolean delete = false;
-  private boolean deleteRobotsNoIndex = false;
-  private boolean deleteSkippedByIndexingFilter = false;
-  private boolean base64 = false;
-  private IndexingFilters filters;
-  private ScoringFilters scfilters;
+  private static boolean skip = false;
+  private static boolean delete = false;
+  private static boolean deleteRobotsNoIndex = false;
+  private static boolean deleteSkippedByIndexingFilter = false;
+  private static boolean base64 = false;
+  private static IndexingFilters filters;
+  private static ScoringFilters scfilters;
 
   // using normalizers and/or filters
-  private boolean normalize = false;
-  private boolean filter = false;
+  private static boolean normalize = false;
+  private static boolean filter = false;
 
   // url normalizers, filters and job configuration
-  private URLNormalizers urlNormalizers;
-  private URLFilters urlFilters;
+  private static URLNormalizers urlNormalizers;
+  private static URLFilters urlFilters;
```

Comment:
Regarding static variables, see comments in Generator. Afaics, they're really shared between IndexerMapper and IndexerReducer. But then the code for initialization is also best shared to avoid duplicate code (cf. setup methods of mapper and reducer).